langmeta/langmeta/shared/src/main/protobuf/semanticdb.proto

```diff
@@ -52,6 +52,7 @@ message Denotation {
   string signature = 3;
   repeated ResolvedName names = 4;
   repeated string members = 5;
+  repeated string override = 6;
```

Comment:
Let's call this `overrides` to be consistent with other collections represented in the schema.